client/src/Movies/Movie.js

```diff
@@ -11,7 +13,7 @@ export default class Movie extends Component {
 
   componentDidMount() {
     // change this line to grab the id passed on the URL
-    const id = 1;
+    const id = this.props.match.params.id;
```

Comment:
Use [destructuring](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#Object_destructuring) over just repeating the same name twice to do the same thing!
```javascript
const { id } = this.props.match.params; 
```